### PR TITLE
Feature/display existing backend

### PR DESCRIPTION
**Pull request by JuusoV**

### CODE REVIEW

**Comment by brylie** [comment]
@elnzv please review this pull request.

File: lib/router/main.js [hunk 1]
```diff
@@ -56,11 +56,25 @@ Router.map(function() {
       return this.subscribe('apiBackend', this.params._id);
     },
     data: function () {
+      var apiBackend = ApiBackends.findOne({_id: this.params._id});
```
Comment (elnzv) [inline lib/router/main.js]: Please, provide some comments here.

File: client/views/api_backends/view_api_backend/view_api_backend.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Template.viewApiBackend.created = function() {
+    var instance = this;
+    var backendId = Router.current().params.apiBackendId;
```
Comment (elnzv) [inline client/views/api_backends/view_api_backend/view_api_backend.js]: Please, provide some comments here.

**Comment by elnzv** [comment]
Looks fine, provide some comments to code and it can be merged.

File: lib/router/main.js [hunk 3]
```diff
@@ -56,11 +56,25 @@ Router.map(function() {
...
+      return { apiBackend: apiBackend }
+    },
+    layoutTemplate: "masterLayout",
+    render: "editApiBackend"
+  })
+
+    this.route("viewApiBackend", {
+    path: "/api/:_id/view",
```
Comment (frenchbread) [inline lib/router/main.js]: More common approach for paths for a case like this think is `/api/:_id` - without `/view`, it is not really needed here once we use `/api/:_id/edit` for editing view.

**Comment by brylie** [comment]
@apinf/developers please review. I have made suggested changes.

**Pull request merged**